### PR TITLE
Don't add certbot ppa in ubuntu 20

### PR DESCRIPTION
**Pull request by benibr**

### CODE REVIEW

**Comment by benibr** [comment]
probably need  `- package_facts: {}`

File: tasks/main.yml [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
+    - ansible_lsb.major_release|int < 20
 
 - name: update apt cache
   apt:
     update_cache: yes
     cache_valid_time: 3600
-  when: ansible_pkg_mgr == "apt"
+  when:
+   - ansible_pkg_mgr == "apt"
+   - ansible_facts.packages["certbot"] is undefined
```
Comment (morbidick) [inline tasks/main.yml]: couldn't you use the same check for the ppa step?

**Comment by morbidick** [comment]
increasing the min ansible version in meta/main.yml should probably be enought?

File: tasks/main.yml (hunk 1 above)
Comment (benibr) [reply tasks/main.yml]: why should we? repo should only be added in ubuntu version 19 and lower.
repo update should only happen if certbot is not already installed, they are two different requirements

**Comment by benibr** [comment]
> increasing the min ansible version in meta/main.yml should probably be enought?

For using package information? As far as I undertand it, package_facts must be called explicitly. Maybe this slows down the run more than the repo updates :-P

https://docs.ansible.com/ansible/latest/modules/package_facts_module.html

**Pull request merged**